test/studies/shootout/submitted/spectralnorm.chpl

```diff
@@ -38,7 +38,7 @@ proc multiplyAv(v: [?Dv], Av: [?DAv]) {
 }
 
 //
-// Compute A-tranpose * v ('Atv').
+// Compute A-transpose * v ('Atv').
```

Comment:
I'd fixed this one in #4136, but it was reverted (deliberately) as part of #6021 so that the ``submitted/`` dir matches what was submitted.